### ISSUE TITLE
myfreemp3juices.cc: nuisance

### ISSUE THREAD

**Issue by Yehh22** [uBlockOrigin/uAssets#17118]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://myfreemp3juices.cc/`

### Category

nuisance

### Description

When I search for a song, it overwrites my clipboard to `Mp3 Download https://myfreemp3.vip`

### Screenshot(s)

N/A

### Configuration

<details>

```yaml
uBlock Origin: 1.47.2
Chromium: 110
filterset (summary): 
  network: 87187
  cosmetic: 41273
  scriptlet: 17380
  html: 0
listset (total-discarded, last-updated): 
  default: 
    user-filters: 0-0, never
    NOR-0: 3140-7, 4m
    easylist: 63813-45, 4m
    easyprivacy: 31035-1175, 4m
    plowe-0: 3756-933, 4m
    ublock-abuse: 78-0, 4m
    ublock-badware: 4681-115, 4m
    ublock-filters: 32813-81, 4m
    ublock-privacy: 289-1, 4m
    ublock-quick-fixes: 359-23, 4m
    ublock-unbreak: 1910-40, 4m
    urlhaus-1: 6564-0, 4m
filterset (user): [empty]
modifiedUserSettings: 
  showIconBadge: false
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 222 ms (selfie)
  maxAssetCacheWait: 152 ms
popupPanel: 
  blocked: 5
  blockedDetails: 
    brocardcored.com: 1
    googletagmanager.com: 1
    iclickcdn.com: 1
    jeeryzest.com: 1
    skimresources.com: 1
```
</details>

**Comment by mapx-** [comment]
It does not work in chrome on my side.

working:
`myfreemp3juices.cc##+js(ra, onclick, [onclick="myFunction()"])`

**Comment by mapx-** [comment]
already tested, does not work

**Comment by garry-ut99** [comment]
Any of the 3 filters works for me on both Firefox & Opera (I don't use Chrome).

**Comment by Daxy7** [comment]
> Any of the 3

are you sure you pressed ctrl+f5 (clears cache on current page) with individual filter enabled while testing?

**Comment by garry-ut99** [comment]
Yes, "100" times and it was working fine - clipboard not overwritten, today I've tried another "100" times and still works fine.
